test/expected_output.md5

```diff
@@ -3,39 +3,39 @@ f1a2c75bdd4dd51c5e7ac6fd04121da3  results/TABLES/counts.miRNA_primary_transcript
-61ad04cdce34d261edfadb05960dc9dc  results/test_lib/oligoGenome_report.txt
+e12475a81bd38bc78abe312c213da3a5  results/test_lib/segemehlGenome_map.sam
+4badb4b34975b4db8d3cb66f87922cda  results/test_lib/catMappings.sam
+cca2905b84063cea544ff53fd39fd3e4  results/test_lib/oligoGenome_converted.sam
+e1436b96948d137f2029e63fb6d9a670  results/test_lib/oligoGenome_report.txt
 6703fa41390a5db74a5f8e5abc1e612e  results/test_lib/noheader_TranscriptomeMappings.sam
-f684aa1780e02e8e11a9743a5721e443  results/test_lib/filtered_for_oligomap.fasta
+03f79d6a18349efde3c39421a27b4310  results/test_lib/filtered_for_oligomap.fasta
 63a32839360a985b68e0685aafad5c54  results/test_lib/fa/reads.fa
 edcb854702519c0002d8ce89a21e54ef  results/test_lib/formatted.fasta
-78e60af8e7c4faa33bd335edc6c594e9  results/test_lib/oligoGenome_sorted.fa
-c98faee4f67aa6b3c0572eed1d14e7b9  results/test_lib/removeInferiors.sam
+867f1b6d7c7d34d08b639651e03c24b4  results/test_lib/oligoGenome_sorted.fa
+cb3d45be34ced8707e61995a2046fb4b  results/test_lib/removeInferiors.sam
 ec0e9bcc8ea857da897035c8fca4078f  results/test_lib/cut.fasta
-e5df13a0b0145b3927979fd95dc502b9  results/test_lib/oligoTranscriptome_report.txt
-b460a93ea47587a34a58d0ac809cd110  results/test_lib/nhfiltered_GenomeMappings.sam
+1a247956e181d5601cea86ce5ed5b2f8  results/test_lib/oligoTranscriptome_report.txt
+7e708cdf835b37ffbab29313abdb6510  results/test_lib/nhfiltered_GenomeMappings.sam
 9f7fa8cf59aea5526de830ba89f9629d  results/test_lib/TranscriptomeMappings.sam
 9f7fa8cf59aea5526de830ba89f9629d  results/test_lib/segemehlTranscriptome_map.sam
-7abcea11d5c13654dc807f7471226ff1  results/test_lib/convertedSortedMappings_test_lib.bam
-d6248adef3e378994e34af83927c5ecd  results/test_lib/header_sorted_catMappings.sam
-2132aa04748dc876b775558a7174000a  results/test_lib/concatenated_header_catMappings.sam
-d29bb846223c027ab89c97236209f549  results/test_lib/alignments.bed12
-052e843614feb3b1be092caca936e8fd  results/test_lib/mappingsConverted.bam
+303295d7212c76ecb09198e3604f0d72  results/test_lib/convertedSortedMappings_test_lib.bam
+1335281f16cf48be32bbe7536aa8a998  results/test_lib/header_sorted_catMappings.sam
+39e068980c856ec82ac3c8965ebd0b58  results/test_lib/concatenated_header_catMappings.sam
+c00b6fc1e30028651a405e32bc48a2f1  results/test_lib/alignments.bed12
+1d355b4071c1a7094cb44d9885eadabd  results/test_lib/mappingsConverted.bam
 d41d8cd98f00b204e9800998ecf8427e  results/test_lib/oligoTranscriptome_map.fa
 9f7fa8cf59aea5526de830ba89f9629d  results/test_lib/nhfiltered_TranscriptomeMappings.sam
 d41d8cd98f00b204e9800998ecf8427e  results/test_lib/TransToGen.sam
-b460a93ea47587a34a58d0ac809cd110  results/test_lib/GenomeMappings.sam
-add9893cbcf01cffa69bec4441684f30  results/test_lib/collapsed.fasta
-ec8a0be25c6f49d580fd3f3d80ece72b  results/test_lib/intersect_mirna.bed
+7e708cdf835b37ffbab29313abdb6510  results/test_lib/GenomeMappings.sam
+7f85f711a7cb605135c8248068a53b92  results/test_lib/collapsed.fasta
```

Comment:
Perhaps this is the most upstream file that changed? You could compare this to a previous output and check what changed. Please report here so that we can decide how to go on.

Reply:
The changes in all the check sums is given by a reorder of the sequences - and therefore the naming - on the `collapsed.fasta` file. Here's a picture of the comparison made. (The left side is the results from this branch, and the left one is from the `dev` one)

![image](https://user-images.githubusercontent.com/103108590/226192004-565bc396-da3f-4baa-8317-22c502e8c436.png)

So, this reorder on the sequences, as the naming is different, makes all the following files to also be different but the final results are the same as the data contained is, in fact,  the same.

Reply:
Okay, thanks a lot. Bit weird that FASTX behaves differently from BioContainers. Anyway, you confirmed that - apart from the order - the collapsed FASTA is identical and that the final results are the same, so I'm fine with changing the checksums :)